Tests/ParseSwiftTests/KeychainStoreTests.swift

```diff
@@ -149,20 +149,20 @@ class KeychainStoreTests: XCTestCase {
     }
 
     func testThreadSafeSet() {
-        DispatchQueue.concurrentPerform(iterations: 100) { _ in
+        DispatchQueue.concurrentPerform(iterations: 50) { _ in
```

Comment:
Is there a reason these are getting turned down?

Reply:
I turned this down originally because it would randomly fail on some tests with 100 threads. I think I fixed the problem with moving the fulfills